packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -217,36 +221,26 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   margin: units(2.5) 0;
   text-align: left;
 
-  thead {
-    th {
-      background-clip: padding-box;
-      color: $table-header-text-color;
-      font-weight: fw("bold");
-      line-height: line-height(
-        $theme-body-font-family,
-        $theme-input-line-height
-      );
-    }
-
-    th,
-    td {
-      background-color: color($theme-table-header-background-color);
-      color: $table-header-text-color;
-    }
-  }
-  tbody {
-    th {
-      text-align: left;
-    }
+  thead td,
+  tfoot td,
+  th {
+    background-clip: padding-box;
+    line-height: line-height($theme-body-font-family, $theme-input-line-height);
+    background-color: color($theme-table-header-background-color);
+    color: $table-header-text-color;
```

Comment:
> [!Note]
> This change relies on the user agent style sheet for `th` to set the font weight to bold. Firefox, Chrome, and Safari all include a default `font-weight:bold` rule for `th`. No action is needed here, just flagging it in case we want to be explicit about the `th` font weight.